### PR TITLE
Update nodejs-6 to 6.2.1

### PR DESCRIPTION
**Pull request by AndWac**

### CODE REVIEW

File: components/runtime/node6/nodejs.p5m [hunk 1]
```diff
@@ -10,23 +10,23 @@
...
-set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
-
-license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
-
-link path=usr/node/6/bin/npm target=../lib/node_modules/npm/bin/npm-cli.js
-link path=usr/bin/node target=../node/6/bin/node mediator=nodejs mediator-version=6
-link path=usr/bin/npm target=../node/6/bin/npm mediator=nodejs mediator-version=6
-link path=usr/share/man/man1/node.1 target=../../../node/6/share/man/man1/node.1 mediator=nodejs mediator-version=6
-
+set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
```
Comment (pyhalov) [inline components/runtime/node6/nodejs.p5m]: Is something wrong with spaces here?

**Comment by AndWac** [comment]
Yes, after I regenerated the file I copied some lines from the old one.
I don't know whether I should blame vim or gnome-terminal...
Should be fixed now.

File: components/runtime/node6/nodejs.p5m [hunk 2]
```diff
@@ -10,7 +10,7 @@
 #
 
 #
-# Copyright 2016 Andreas Wacknitz
+# Copyright 2016 Andreas Wacknitz 
 #
```
Comment (pyhalov) [inline components/runtime/node6/nodejs.p5m:14]: sorry, same here :)

**Comment by AndWac** [comment]
Sigh, should be fixed now.
The good thing is I improve my git abilities :-)

**Pull request merged**